### DISCUSSION TITLE
Where to start!?

### DISCUSSION THREAD

**Discussion by Drakemoor** [brianpetro/obsidian-smart-connections#446] [Q&A] [answered]
I would like to try this, but I have no idea where to start. I have no idea what the settings mean, nor which or what I have to change to make it work. 

Do I need to learn all the terminology to use it? 

Is there a way to use the plugin without paying for GPT? 

I would appreciate any help.

**Comment by brianpetro** [comment, marked as answer]
Hi @Drakemoor 

Thanks for checking out Smart Connections.

You can use the local embedding models and Smart View without any OpenAI API key.

The default settings, BGE-micro for block embeddings, should be all you need. Just open the Smart View using Obsidian's command palette, click "Start Embedding" in the notification when prompted, and you should be good to go.

🌴
